user-guide/modules/ROOT/pages/task-simulation.adoc

```diff
@@ -20,10 +24,365 @@ Creating a real-time simulation of vehicles involves various aspects, including
  
 * boost:graph[]: In case you need to represent roads or pathways as a graph, this library provides a flexible and powerful way to represent and manipulate graphs. It also includes a number of graph algorithms.
 
-* boost:thread[] or boost:asio[]: To achieve real-time performance, you might need to make use of multi-threading or asynchronous input/output. boost:thread[] provides classes and functions for multi-threading, synchronization, and inter-thread communication. boost:asio[] is a cross-platform library for asynchronous programming and can handle a lot of networking tasks as well.
+* boost:thread[] or boost:asio[]: To achieve real-time performance, you might need to make use of multi-threading or asynchronous input/output. boost:thread[] provides classes and functions for multi-threading, synchronization, and inter-thread communication, and would be useful to parallelize calculations for physics updates. boost:asio[] is a cross-platform library for asynchronous programming and can handle a lot of networking tasks as well.
```

Comment:
I'm not sure we should keep talking about libraries like Boost.Thread in these contexts where we describe what makes Boost unique. This functionality has been available as `std::thread` in the standard library for 13 years, and I don't think Boost.Thread has received much maintenance or new features since then.